<?xml version="1.0"?>
<launch>
  <node pkg="opencv_ros" name="opencv_camera" type="camera_pub" output="screen"/>
            <param name="image_view" type="bool" value="true"/>
             <param name="video_device" type="int" value="0"/>
</launch>

